app/src/main/java/com/rahobbs/todo/helpers/TodoItem.java

```diff
@@ -0,0 +1,88 @@
+package com.rahobbs.todo.helpers;
+
+import java.util.Date;
+import java.util.UUID;
+
+/**
+ * Created by rachael on 2/1/16.
+ * Model for a single to-do item
+ */
+public class TodoItem {
+    private String mTitle;
+    private String mDetails;
+    private Date mDate;
+    private UUID mID;
+    private boolean completed;
```

Comment:
personally, I'm not a fan of Hungarian notation (the prepending everything with `m`) but that's a personal preference. 

but if you are going to use, just be consistent with it :+1: